cafebabel/templates/home.html

```diff
@@ -24,19 +25,23 @@
 
 {% block body_class %}home{% endblock %}
 {% block body %}
-{% if articles %}
-  {{ card(articles|first, image_dimensions=config.RESIZE_SIZE_ARTICLE_COVER) }}
-{% endif %}
-
-  <h2>Sections</h2>
-{% for category in categories|shuffle %}
-  <a href={{ category.detail_url }} class="section-link {{ category.slug }}">{{ category.name }}</a>
-{% endfor %}
-
-{% if articles|length > 1 %}
-  <h2>Recently published</h2>
-  {% for article in articles[1:] %}
-    {{ card(article) }}
-  {% endfor %}
-{% endif %}
+  <section class=editors-pick>
+    {% set tag_editors_pick, articles_editors_pick = articles_for_tag('editors-pick', limit=1) %}
+    {% for article_editors_pick in articles_editors_pick %}
+      {{ card(article_editors_pick, image_dimensions=config.RESIZE_SIZE_ARTICLE_COVER) }}
+    {% endfor %}
+  </section>
+  <section class=categories>
+    <h2>Sections</h2>
+    {% for category in categories|shuffle %}
+      <a href={{ category.detail_url }} class="section-link {{ category.slug }}">{{ category.name }}</a>
+    {% endfor %}
+  </section>
+  {{ latest("latest1", 3) }}
+  {{ series("meet-my-hood", 2, "description lalala") }}
+  {{ latest("latest2", 3) }}
```

Comment:
All these "latest1"/"latest2" really look like tweaking and hacking a system that is barely being born :(
I guess we have miscomunicated on the need here.
Too late to fix I guess